src/quo2/components/wallet/account_card/style.cljs

```diff
@@ -102,3 +111,25 @@
 
 (def metrics-icon-container
   {:margin-left 4})
+
+(defn gradient-start-color
+  [theme]
+  (colors/theme-colors
+   (colors/custom-color :army 50 0)
```

Comment:
these should be using `customization-color` prop and not fixed `:army` key

Reply:
Done